frontend/public/style/_overrides.scss

```diff
@@ -135,6 +135,18 @@ h6 {
   padding-right: 0;
 }
 
+.pf-c-button.co-p-has-sidebar__sidebar-close {
+  float: right;
+  font-size: 21px !important;
+  line-height: 1;
+  opacity: .35;
+  padding: 0;
+  &:hover,
+  &:focus {
+    opacity: 1;
+  }
+}
+
```

Comment:
Now included in #2889, so please drop the changes in this PR.

Reply:
dropped